domains/Settings/SupportHome/SupportSettings/Feedback/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import {
+  View
+} from 'react-native';
+import email from 'react-native-email';
```

Comment:
hmm this library isn't the most supported, I think you mentioned that you were trying to use [expo mail composer right?](https://docs.expo.io/versions/latest/sdk/mail-composer/). Why didn't you use that again?